content/docs/intro/console/collaboration/auditing.md

```diff
@@ -28,19 +28,19 @@ of the call the user made.
 ## Viewing Audit Logs in the Console
 
 Audit logs are available to organizations with an Enterprise level subscription
-only. If you are an organization administrator, you can view your organization's audit logs,
-by selecting your organization from the organization drop down. Then click on the settings tab.
-On the left nav-bar you should see a tab called Audit Logs. Audit Logs allow you to view
+only. If you are an organization admin, you can view your organization's audit logs,
+by selecting your organization from the organization drop down. Then select the **Settings** tab.
+On the left nav-bar you should see a tab called **Audit Logs**. Audit Logs allow you to view
```

Comment:
@davidwrede What's our story on the **bolding** thing? Is this an intentional change? Up to now, we've rarely used bold in prose like this (rarely used it at all in fact), and instead just relied on capitalization, mainly to keep the visual noise level down (and I'm generally a fan of less boldness for this reason -- it's usually just distracting). We should probably clarify one way or another, though, so we can get consistent from here on out. 

(Incidentally my vote would be for no-bold. 😇 🙏 In addition to just looking calmer and being equally clear, it also makes it easier to call attention to the things that really warrant it.)

Reply:
Yes, I'd like us to be intentional about bolding important UI elements. This is generally a best practice when documenting steps or items in a UI (e.g. both [Google's developer guide](https://developers.google.com/style/ui-elements#formatting) and the [MSTP style guide](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions#in-the-ui-and-general-content) use this convention).

If the bolding itself is too heavy weight, we should probably revisit the font and styling we use on docs to collectively thin the text out a bit.

Reply:
lets wait until we have a new font, then we can figure out the weight to use for ui elements.